Homework7/Unit.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework7
+{
+    abstract class Unit
+    {
+        protected Unit(double ad, string name)
+        {
+            this.Ad = ad;
+            this.Name = name;
+        }
+
+        public double Hp { get; set; }
```

Comment:
`public set`
Do you think that it's good idea?

Reply:
fixed